pkg/qemu/qemu.go

```diff
@@ -194,8 +196,19 @@ func Cmdline(cfg Config) (string, []string, error) {
 	args = append(args, "-cdrom", filepath.Join(cfg.InstanceDir, filenames.CIDataISO))
 
 	// Network
+	if y.Network.VDE.SocketDir != "" {
+		macAddr := y.Network.VDE.MACAddress
+		if macAddr == "" {
+			sha1 := sha1.Sum([]byte(cfg.InstanceDir))
```

Comment:
Any reason not to use sha256?

Reply:
Also, can we split function?

Reply:
No reason. It doesn't really matter, as it is not a security thing. I'll change it.